installing.rst

```diff
@@ -293,42 +288,58 @@ If you have Gauge and your language runner installed, then see how you can :ref:
 Uninstallation
```

Comment:
Uninstallation can have a section telling users that the `.gauge` folder which also has `config` is not removed, and has to be removed manually.